hw-5/script.js

```diff
@@ -0,0 +1,154 @@
+$(document).ready(function(){
+	//set up page to clear placeholder text in search field on focus, and attach "callAPI" event to search button
+	$('#userinput').focus(function() {
+		$(this).val('');
+		});
+
+	$('#searchenter').click(function() {
+		callAPI($('#userinput').val());
+	});
+});
+
+
+
+// Event hander for calling the SoundCloud API using the user's search query
+function callAPI(query) {
+	// Remove any existing Stratus player with song from previous search
+	$('#stratus').remove();
+
+	$.get("https://api.soundcloud.com/tracks?client_id=b3179c0738764e846066975c2571aebb",
+		{'q': query,
+		'limit': '200'},
+		function(data) {
+			showResults(data);
+			//console.log(data);
+		},'json'
+	);
+}
+
+function showResults (responseObject) {
```

Comment:
There is a lot of repetitive code in this function. I think a better pattern would have been to break up the actual rendering of the elements to a separate function that is called by this function. Basically, any code that renders to the DOM should be contained within a small function, usually referred to as the renderer. Check out this post on separating the concerns in your code by using the MVC pattern: https://alexatnet.com/articles/model-view-controller-mvc-javascript.